src/content/app/tools/blast/views/blast-form/BlastForm.scss

```diff
@@ -31,3 +31,20 @@
   display: flex;
   flex-direction: column;
 }
+
+.blastFormStepToggle {
+  display: flex;
+  justify-content: flex-end;
+  gap: 10px;
+  max-width: 1790px;
```

Comment:
Is this a magic number related to other magic numbers on the BLAST pages? If yes, could you leave a TODO comment saying that we need to group those magic numbers into variables? See all the TODOs in the merged PR https://github.com/Ensembl/ensembl-client/pull/745

There is https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1602 which would make a very nice addition to sprint 9. It's about cleaning up the code. It doesn't specifically talk about all these shared widths on BLAST pages. We probably need a dedicated ticket for that; would also make a good addition to sprint 9.